sanssouci/post_hoc_bounds.py

```diff
@@ -93,6 +93,35 @@ def min_tp(p_values, thr):
     return p_values.shape[0] - max_fp(p_values, thr)
 
 
+def min_tdp(p_values, thr):
+    """
+    Lower bound for the proportion of true discoveries in a selection
+
+    Parameters
+    -----------
+
+    p_values : 1D numpy.array
+        A 1D numpy array of p-values for the selected items
+    thr : 1D numpy.array
```

Comment:
```suggestion
    threshold : 1D numpy.array
```

Reply:
Good point but for consistency with the rest of the code, it should be addressed separately. Cf #18